java/query/query-core/src/main/java/sleeper/query/core/output/ResultsOutput.java

```diff
@@ -20,9 +20,18 @@
 import sleeper.query.core.model.QueryOrLeafPartitionQuery;
 
 /**
- * An interface that classes for sending the results of queries somewhere must implement.
+ * Results output.
```

Comment:
This is just the interface name. The previous version explained a little about what it was for.